tests/components/sensibo/test_climate.py

```diff
@@ -80,6 +81,7 @@ async def test_climate(
     caplog: pytest.LogCaptureFixture,
     get_data: SensiboData,
     load_int: ConfigEntry,
+    snapshot: SnapshotAssertion,
```

Comment:
Test refactoring should not be in the PR that bumps a dependency.

Please separate this.